src/monodroid/jni/monodroid-glue.c

```diff
@@ -3361,13 +3361,28 @@ set_profile_options (JNIEnv *env)
 	if (!output) {
 		const char* col = strchr (value, ':');
 		char *ovalue;
+		char *extension;
 
-		output = path_combine (override_dirs [0], "profile.mlpd");
+		if (col && !strncmp (value, "log:", 4) || !strncmp (value, "log", 3))
```

Comment:
Why check for both "starts-with" `log:` *or* `log`? If the value is `log-me-for-greater-good`, should we still be using `.mlpd`?

Also, I'm not C-wizard-enough to know the precedence between && and || here. I would prefer some parenthesis for clarity.

Reply:
We should check for "log" case too, your note is right though and we should use plain strcmp for that case.

The C precedence is the same as in C#. I will add parenthesis. I am not used to it as I earlier worked in a project where there were considered unnecessary and the style guide requested to not use them where not needed :-)